### DISCUSSION TITLE
Handover feature

### DISCUSSION THREAD

**Discussion by kczaplin** [aligungr/UERANSIM#492] [Q&A]
Hello,
Do you have any update on handover? When do you plan to implement and release this feature?

Regards

**Comment by aligungr** [comment]
Hello

Unfortunately there is no update on handover feature.
I'm a bit sorry to say that, but handover feature should **not** be expected in the near feature.

BR

**Reply by cin89** [reply]
Hi,

Is there any plan to add possibility of creating more than one cell on gNB?

BR

**Reply by aligungr** [reply]
@cin89

Indeed you can use more than one gNodeBs. However you need to bind each gNodeB to a single network adapter.

In other words, you have to use different IP addreses.
